src/CronTask.php

```diff
@@ -2115,13 +2115,31 @@ public static function callCronForce()
         /** @var array $CFG_GLPI */
         global $CFG_GLPI;
 
-        $path = $CFG_GLPI['root_doc'] . "/front/cron.php";
+        if (self::mustRunWebTasks()) {
+            $path = $CFG_GLPI['root_doc'] . "/front/cron.php";
+            echo "<div style=\"background-image: url('$path');\"></div>";
+        }
 
-        echo "<div style=\"background-image: url('$path');\"></div>";
         return true;
     }
 
 
+    /**
+     * Check if any web cron task exist and is enabled
+     *
+     * @return bool
+     **/
+    protected static function mustRunWebTasks(): bool
+    {
+        $web_tasks_count = countElementsInTable(self::getTable(), [
+            'mode'  => 1, // "GLPI" mode
+            'state' => ['<>', 0], // State is not "disabled"
```

Comment:
Only waiting tasks will be executed, so if there are only "running" tasks, there is no need to call the cron either.

```suggestion
            'state' => 1, // State is "waiting"
```